backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -446,17 +452,26 @@ def delete_for_ide(self, request: HttpRequest, pk: uuid) -> Response:
         document_id: str = serializer.validated_data.get(
             ToolStudioPromptKeys.DOCUMENT_ID
         )
+        org_id = UserSessionUtils.get_organization_id(request)
+        user_id = custom_tool.created_by.user_id
```

Comment:
Did you tested in OSS. I think we can use `id` instaed of `user_id` . `user_id` might be used for thirt party authentication.

Reply:
`document_indexing:mock_org::doc_id` This is how key will be in OSS. Will this be a problem as we only have one user model in OSS? Cascade delete for redis key also works.

Reply:
Are you asking as to whether polling can be moved to FE? Moving polling to FE is not possible.

Reply:
@chandrasekharan-zipstack The User model is present in OSS, but the user_id field serves a different purpose than user.id. The user_id field is intended for third-party authentication IDs, whereas id is the default primary key and applies to OSS as well.